include/dpp/channel.h

```diff
@@ -626,13 +626,21 @@ class DPP_EXPORT channel : public managed, public json_interface<channel> {
 
 	/**
 	 * @brief Add permission overwrites for a user or role.
-	 * If the channel already has permission overwrites for the passed target, the existing ones will be adjusted by the passed permissions
+	 * If the channel already has permission overwrites for the specified target, the existing ones will be adjusted by the passed permissions.
+	 * Any permission flags that are not passed to the `allowed_permissions` or `denied_permissions` will remain unchanged.
```

Comment:
this doesnt read right as it's a bitmask, you pass all of them every time in the api call not a subset of bits, so its not possible to "leave some unchanged" apart from via OR/AND/NOT

Reply:
aight, reverted it.